frontend/packages/kubevirt-plugin/src/components/vms/vm-resource.tsx

```diff
@@ -93,6 +94,10 @@ export const VMDetailsList: React.FC<VMResourceListProps> = ({ vm, vmi, pods, mi
           DASH
         )}
       </dd>
+      <dt>CD</dt>
+      <dd id={prefixedID(id, 'cd-rom')}>
+        {sortedCds.length > 0 ? <BootOrder bootableDevices={sortedCds} /> : 'No Disk'}
```

Comment:
@rawagner seems reasonable to use `BootOrder` to display the multiple CDs in boot order?

Reply:
yes, it does!

But we are trying to get rid of kubevirt-web-ui-components dependencies so I'd suggest to port `BootOrder` and `getBootableDevicesInOrder` to `kubevirt-plugin`.

I personally would expect that `BootOrder` will order the devices itself (it will call `getBootableDevicesInOrder` on `bootableDevices` prop)